web/src/open-dialog.js

```diff
@@ -138,6 +138,7 @@ export class OpenDialog extends EventTarget {
     input.name = 'torrent-files[]';
     input.id = input_id;
     input.multiple = true;
+    input.accept = ".torrent";
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/input/file#accept

Maybe also add mime type for compatibility?


```suggestion
    input.accept = ".torrent,application/x-bittorrent";
```

Reply:
Ok, added